src/main/java/eu/fasten/vulnerabilityproducer/utils/mappers/PurlMapper.java

```diff
@@ -42,24 +42,23 @@
 
     public PurlMapper(VersionRanger versionRanger, PatchFinder patchFinder, String pathToMaps) {
         this.versionRanger = versionRanger;
-        this.patchFinder = patchFinder;
-        this.mavenMap = loadReposMapFromMemory(pathToMaps + "repo_map_maven.json");
-        this.pypiMap = loadReposMapFromMemory(pathToMaps + "repo_map_pypi.json");
-        this.cpeMap = loadReposMapFromMemory(pathToMaps + "cpe_map_repos.json");
+        this.patchFinder   = patchFinder;
```

Comment:
Do Java devs also follow this convection of aligning variables by the equal op.? :)

Reply:
I have not seen this done in Java but rather in Haskell. I feel like it improves the readability of the code.